cluster-provision/k8s/1.21/provision.sh

```diff
@@ -4,7 +4,6 @@ set -ex
 
 # Configure cgroup version
 if [ "${UNIFIED_CGROUP_HIERARCHY}" == "1" ]; then
-    CMDLINE_LINUX_APPEND="${CMDLINE_LINUX_APPEND} systemd.unified_cgroup_hierarchy=1"
```

Comment:
Probably the same line can be dropped from `cluster-provision/k8s/*1.20*/provision.sh` as well